qml/WebsiteDisplayPage.qml

```diff
@@ -26,8 +26,8 @@ BasePage {
 
     customBack: true
 
-    WebView {
-        id: webView
+    WebEngineView {
```

Comment:
Is that on purpose that we don't use LuneOS.Components ? If you don't see any reasons not to do so, I'd prefer we use a unified way of showing web content, so using LunaWebEngineView